logging/log.py

```diff
@@ -4,9 +4,10 @@
 
 class Log:
 
-    def __init__(self, log_file_path, log_name, log_units, params, increment=True):
```

Comment:
Can we add a test to the test suite to show that this works as intended? What would it look like?